src/ikonScraperInterface.py

```diff
@@ -209,18 +209,34 @@ def checkForOpenings(driver, datesAvailable):
 					vals = (month, day, year)
 					cursor.execute(sql, vals)
 					# if not, insert into db send email alert
-					if cursor.rowcount != 0:
-						reserveDay(driver, monthsToCheck[month], day, year, mountain)
-						# return to make reservation page
-						url = "https://account.ikonpass.com/en/myaccount/add-reservations/"
-						driver.get(url)
-						# get day of week
-						dayOfWeek = datetime.date(year, month, day).strftime("%A")
-						# send alert
-						emailInterface.sendDateToReserveAlertEmail("jjohnson11096@gmail.com", mountain, monthsToCheck[month], str(day), str(year), dayOfWeek, ikonEmail)
-						# refresh scraper
-						selectMountain(driver, mountain)
-						selectMonth(driver, monthsToCheck[month], year)
+					row = cursor.fetchone()
+					if row:
+						# check if current mountain and ikon email is valid for this date.
+						# empty mountain/ikon email in db means all values are valid.
+						shouldReserve = True
+						(dbMonth, dbDay, dbYear, dbMountainsAsStr, dbIkonEmailsAsStr) = row
+						if dbMountainsAsStr:
+							dbMountains = dbMountainsAsStr.split(',')
+							if mountain not in dbMountains:
```

Comment:
I generally don't like having variables with the type name in them. Would it work if you just did: if mountain not in dbMountainsAsStr.split(','), but rename it to just dbMountains